### DISCUSSION TITLE
How to select 720 quality video to load?

### DISCUSSION THREAD

**Discussion by assopri** [omansak/libvideo#215] [Q&A]
Hi! Be default getting 360 resolution. How to get HD (720)? My eventual goal is to load the best quality version, available with the sound.

**Comment by assopri** [comment]
Solution found. Thank you!

**Reply by prafulravindra** [reply]
What is the solution? Even I am looking for the same, need help.

var videoInfos = youTube.GetAllVideos(link);
var maxResolution = videoInfos.FirstOrDefault(v => v.Resolution == 1080);

I have set the max resolution as '1080p' but how do I use it with 'GetAllVideos' to save the video with this resolution.
If I use 'GetVideo' then downloaded video is defaulted with '360p' resolution.

![image](https://user-images.githubusercontent.com/17843849/144998993-1a0576bf-207d-40d8-a008-b1cf7bdfb88f.png)
